crds/jwst/specs/miri_pars-darkpipeline.rmap

```diff
@@ -1,17 +1,23 @@
 header = {
-    'derived_from' : 'Notebook generated by J.Eisenhamer 2020-10',
+    'classes' : ('Match', 'UseAfter'),
+    'derived_from' : 'Hand-made, SCSB, CRDS-416, 2020-10',
     'file_ext' : '.asdf',
     'filekind' : 'pars-darkpipeline',
     'filetype' : 'pars-darkpipeline',
     'instrument' : 'MIRI',
     'mapping' : 'REFERENCE',
-    'name' : 'jwst_miri_pars-darkpipeline.rmap',
+    'name' : '',
     'observatory' : 'JWST',
-    'parkey' : (('META.EXPOSURE.TYPE',), ('META.OBSERVATION.DATE', 'META.OBSERVATION.TIME')),
-    'sha1sum' : '0d673dc07351bf25f5b1bda9aa60c959aeb7568d',
+    'parkey' : ((), ('META.OBSERVATION.DATE', 'META.OBSERVATION.TIME')),
+    'sha1sum' : '4c1c61034a2c5f3726a01c53b026804ee6532ae8',
     'suffix' : 'pars-darkpipeline',
     'text_descr' : 'DarkPipeline runtime parameters',
 }
 
+comment = """
+Step parameter references are JWST step and pipeline configurations.
+This reftype provides the configuration for jwst.pipeline.DarkPipeline
+"""
+
 selector = Match({
```

Comment:
If you do opt to keep the default match rule, does this selector need one?

Reply:
removing the defaults, so non-issue.